packages/frontend/src/components/SchedulersView/LogsView.tsx

```diff
@@ -70,11 +70,21 @@ const Logs: FC<LogsProps> = ({
         [openedUuids],
     );
 
-    const groupedLogs = useMemo(
-        () => Object.entries(groupBy(logs, 'jobGroup')),
-        [logs],
-    );
-
+    const groupedLogs = useMemo(() => {
+        // Get only last log per job
+        // Logs should already by sorted by status and time
+        const uniqueLogs = logs.reduce<SchedulerLog[]>((acc, log) => {
+            if (
+                acc.some(
+                    (l) => l.jobGroup === log.jobGroup && l.task === log.task,
```

Comment:
should this be filtered in the SQL query? So we only get the latest task state

Reply:
I thought about that first, but I was afraid this was used somewhere else... but it seems `logs` are only used here, so I think we can do it

Reply:
mmm, I'm not doing it in the sql, but I'm doing it in the model, after parsing and sorting, what do you think ?